config.w32

```diff
@@ -153,10 +153,20 @@ if (PHP_MONGODB != "no") {
     mongoc_ssl_path_to_check += ";" + PHP_OPENSSL;
   }
 
-  if (CHECK_LIB("ssleay32.lib", "mongodb", mongoc_ssl_path_to_check) &&
-      CHECK_LIB("libeay32.lib", "mongodb", mongoc_ssl_path_to_check) &&
-      CHECK_LIB("crypt32.lib", "mongodb") &&
-      CHECK_HEADER_ADD_INCLUDE("openssl/ssl.h", "CFLAGS_MONGODB")) {
+  var mongoc_ssl_found = false;
+
+  /* PHP 7.1.2 introduced SETUP_OPENSSL(), which supports OpenSSL 1.1.x. Earlier
+   * versions will use the legacy check for OpenSSL 1.0.x and lower. */
+  if (typeof SETUP_OPENSSL === 'function') {
+    mongoc_ssl_found = SETUP_OPENSSL("mongodb", mongoc_ssl_path_to_check) > 0;
+  } else if (CHECK_LIB("ssleay32.lib", "mongodb", mongoc_ssl_path_to_check) &&
+             CHECK_LIB("libeay32.lib", "mongodb", mongoc_ssl_path_to_check) &&
+             CHECK_LIB("crypt32.lib", "mongodb", mongoc_ssl_path_to_check) &&
```

Comment:
I don't know how the windows pecl infrastructure works, but these are not the names of the openssl 1.1.0 symbols.

Is it problematic to vendor in SETUP_OPENSSL() ? That way, people can still use openssl 1.1.x with php7.0 and php5.6

Reply:
These are only the OpenSSL 1.0.x libraries. I noted in the code comment above these conditions:

> Earlier versions will use the legacy check for OpenSSL 1.0.x and lower.

I considered copying the logic of `SETUP_OPENSSL()` into our `config.w32`, but thought that may create a situation where PHP and the OpenSSL extension link 1.0.x but we end up with 1.1.x.